### ISSUE TITLE
config.on is not a function

### ISSUE THREAD

**Issue by TooAngel** [ScreepsMods/screepsmod-mongo#2]
When starting the server I run into the issue: `Error loading "/home/tw/tooangel/screeps-server/node_modules/screepsmod-mongo/index.js": TypeError: config.on is not a function`

`Server version 2.7.2`
`screepsmod-mong 1.0.0`
`nodejs v7.10.1`

```
==> logs/1501674061811/backend.log <==
Loading mods from "/home/tw/tooangel/screeps-server/mods.json"
Error loading "/home/tw/tooangel/screeps-server/node_modules/screepsmod-mongo/index.js": TypeError: config.on is not a function
    at module.exports (/home/tw/tooangel/screeps-server/node_modules/screepsmod-mongo/lib/backend/api/index.js:11:10)
    at module.exports (/home/tw/tooangel/screeps-server/node_modules/screepsmod-mongo/lib/backend/index.js:2:19)
    at module.exports (/home/tw/tooangel/screeps-server/node_modules/screepsmod-mongo/lib/index.js:3:43)
    at modsJson.mods.forEach.file (/home/tw/tooangel/screeps-server/node_modules/@screeps/common/lib/config-manager.js:37:21)
    at Array.forEach (native)
    at Object.exports.load (/home/tw/tooangel/screeps-server/node_modules/@screeps/common/lib/config-manager.js:29:23)
    at Object.module.exports.start (/home/tw/tooangel/screeps-server/node_modules/@screeps/backend/lib/index.js:14:26)
    at Object.<anonymous> (/home/tw/tooangel/screeps-server/node_modules/@screeps/backend/bin/start.js:2:25)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
```

**Comment by ags131** [comment]
Fixed in 47d36e9 Thanks!

**Comment by TooAngel** [comment]
Well, yes, fixed the api require on my machin, too. Could have sent a PR. The `config.on` error appears after fixing the `require`
